packages/core/src/hydration/error_handling.ts

```diff
@@ -101,11 +101,18 @@ export function validateSiblingNodeExists(node: RNode|null): void {
 /**
  * Validates that a node exists or throws
  */
-export function validateNodeExists(node: RNode|null): void {
+export function validateNodeExists(
+    node: RNode|null, lView: LView|null = null, tNode: TNode|null = null): void {
   if (!node) {
-    throw new RuntimeError(
-        RuntimeErrorCode.HYDRATION_MISSING_NODE,
-        `Hydration expected an element to be present at this location.`);
+    const header =
+        'During serialization, Angular expected an element to be present at this location.\n\n'
```

Comment:
nit: this error is thrown on the client
```suggestion
        'During hydration, Angular expected an element to be present at this location.\n\n'
```